src/hotspot/share/compiler/oopMap.hpp

```diff
@@ -112,7 +112,7 @@ class OopMapValue: public StackObj {
   void write_on(CompressedWriteStream* stream) {
     stream->write_int(value());
     if(is_callee_saved() || is_derived_oop()) {
-      stream->write_int(checked_cast<int>(content_reg()->value()));
```

Comment:
int --> juint, we need checked_cast<juint> here because of -Wsign-conversion

Reply:
The result of `content_reg()->value()` is of type `int`, making this checked_cast unnecessary.  Dealing with
possibly missing checked_casts to suppress warnings is out of scope for this PR.